packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -264,6 +329,199 @@ contract LockedGold is
     return totalNonvoting;
   }
 
+  /**
+   * Delegates CELO to delegatee.
+   * @param delegatee The delegatee account.
+   * @param percentageToDelegate Percents of total CELO that will be delegated from delegatee.
+   */
+  function delegateGovernanceVotes(address delegatee, uint256 percentageToDelegate) external {
+    require(
+      percentageToDelegate > 0 && percentageToDelegate <= 100,
+      "delegated percents can be only between 1%..100%"
+    );
+    address delegatorAddress = getAccounts().voteSignerToAccount(msg.sender);
+    address delegateeAccount = getAccounts().voteSignerToAccount(delegatee);
+
+    Delegated storage delegated = delegatorInfo[delegatorAddress];
+    delegated.delegatees.add(delegateeAccount);
+    require(delegated.delegatees.length() <= maxDelegateesCount, "Too many delegatees");
+
+    DelegatedInfo storage currentDelegateeInfo = delegated
+      .delegateesWithPercentagesAndAmount[delegateeAccount];
+
+    require(
+      percentageToDelegate >= currentDelegateeInfo.percentage,
+      "Use revokeDelegatedGovernanceVotes to decrease delegated amount."
+    );
+
+    uint256 requestedToDelegate = delegated
+      .totalDelegatedCeloInPercents
+      .sub(currentDelegateeInfo.percentage)
+      .add(percentageToDelegate);
+
+    require(requestedToDelegate <= 100, "Cannot delegate more than 100%");
+
+    uint256 totalLockedGold = getAccountTotalLockedGold(delegatorAddress);
+    require(totalLockedGold > 0, "No locked celo");
```

Comment:
I don't know if I live this check, a multisig or protocol may want to choose what their delegation may look like before they actually have the Celo.

Reply:
Updated